pkg/nsx/util/utils.go

```diff
@@ -270,13 +283,13 @@ func HandleHTTPResponse(response *http.Response, result interface{}, debug bool)
 		return nil, nil
 	}
 
-	if debug {
-		log.V(2).Info("Received HTTP response", "response", string(body))
-	}
 	if err := json.Unmarshal(body, result); err != nil {
 		log.Error(err, "Failed to convert HTTP response to result", "result type", result)
 		return err, body
 	}
+	if debug {
```

Comment:
since we have debug control inside dumpResponseBody, there is no need to have this extra argument.
I suggest remove this parameter.

Reply:
It's strange to have a parameter to controller debug behavior instead of config option or log level option.

Reply:
it's special for JWT token, when the function used for JWT, it should not output the body